resources/views/users/print.blade.php

```diff
@@ -242,7 +241,15 @@
         </table>
     @endif
 
+    @php
+    $currenttime = Helper::getFormattedDateObject(now(), 'date', false);
+    @endphp
+
     <br>
+    <br>
+    //needs to fix string/int error
+    <div style='text-align:left'><div style='color:black'>{{ trans('admin/users/general.all_assigned_list_generation')}} {{ date('d-M-Y H:i', $currenttime) }}
```

Comment:
In general, we try not to inject PHP into blades where we can help it, especially to create a one-off variable that we don't use anywhere else in the blade. Also if we're using the `Helper::getFormattedDateObject()`, that helper method is going to format it as the user wants it to appear. When you use the `date()` formatter, you're going to override that. 

```php
{{ Helper::getFormattedDateObject(now(), 'date', false) }}
```

should suffice IMHO